src/style.css

```diff
@@ -1,3 +1,82 @@
 body {
   margin: 0;
+  padding: 0;
```

Comment:
`padding: 0;` is not necessary for body. Remove it

Reply:
removed